runtime/str.go

```diff
@@ -157,6 +157,23 @@ func strAdd(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return NewStr(stringV + stringW).ToObject(), nil
 }
 
+func strCapitalize(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	expectedTypes := []*Type{StrType}
+	if raised := checkMethodArgs(f, "capitalize", args, expectedTypes...); raised != nil {
+		return nil, raised
+	}
+	s := toStrUnsafe(args[0]).Value()
+	buf := bytes.Buffer{}
+	numBytes := len(s)
+	if numBytes > 0 {
+		buf.WriteRune(unicode.ToUpper(rune(s[0])))
```

Comment:
Given that Go does support the same notion of locales, then I don't see much we can do for now other than list it as a known limitation somewhere.  Maybe things will change in the future, but I think a partial implementation with a documented limitation is a good start (better than not implementing the method at all).

I will look into limiting the conversions to `0 <= b < 128`.  Thanks!